components/Map/zoomTooltip.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tooltip from '@mui/material/Tooltip';
+
+function ZoomTooltip({ show }) {
+  return (
+    <Tooltip
+      placement="top-end"
+      arrow
+      open={show}
+      title={(
+        <div>
+          <p>
+            <strong>
+              Zoom features are limited while locked into a
+              neighborhood council.
+            </strong>
+            <br />
+            To reset zoom features, please exit by clicking out of the
+            selected area.
+          </p>
+        </div>
+      )}
+      //* changing styles here changes the color of the zoom control, not the tooltip
+    >
+      {/* empty span for positioning the zoomtooltip */}
+      <span
+        className="mapboxgl-ctrl-icon minus-sign-clone"
```

Comment:
So smart! Nothing to do here, I just like this over my implementation. You can mark this as resolved 😄